docs/pages/v2/crm-v2/api-reference/batch-get-accounts.mdx

```diff
@@ -13,15 +13,13 @@ curl --request POST \
   --data '{
   "include": { $native: false },
   "ids": ["123123", "456456", "5675675"],
-  "cursor": "123123",
   "associations": [
     "deals",
     "contacts",
     "leads"
   ]
 }'
 ```
-
 </RequestExample>
 
-
+*Note:* The maximum number of ids per request is 200.
```

Comment:
should we add a sentence about what to do if they need to paginate?

Reply:
I added some code as well, let me know what you think